File: algorandsamples/mypythondemo/Indexer/SearchTransactionsMinAmount.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer
data = {
    "indexer_token": "",
    "indexer_address": "http://localhost:8980"
}
# instantiate indexer client
myindexer = indexer.IndexerClient(**data)
data = {
    "min_amount": 10
}
response = myindexer.search_transactions(**data)
# Pretty Printing JSON string 
print(json.dumps(response, indent=2, sort_keys=True))

File: algorandsamples/v2/myjsdemo/indexer/searchTransactionsMinAmount.js
// SearchTransactionsMinAmount.js
// requires algosdk@1.6.1 or higher 
// verify installed version
// npm list algosdk

const algosdk = require('algosdk');
// const indexer_token = "";
// const indexer_server = "http://localhost";
// const indexer_port = 8980;
const indexer_server = "https://testnet-algorand.api.purestake.io/idx2/";
const indexer_port = "";
const indexer_token = {
    'X-API-key': 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab',
}
// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);

(async () => {
    let currencyGreater = 10;
    let transactionInfo = await indexerClient.searchForTransactions()
        .currencyGreaterThan(currencyGreater).do();
    console.log("Information for Transaction search: " + JSON.stringify(transactionInfo, undefined, 2));
})().catch(e => {
    console.log(e);
    console.trace();
});